app/helpers/decidim/challenges/application_helper.rb

```diff
@@ -5,6 +5,23 @@ module Challenges
     # Custom helpers, scoped to the challenges engine.
     #
     module ApplicationHelper
+      include PaginateHelper
```

Comment:
can't we add it to the controllers that really need it as a helper instead of in the ApplicationHelper?